Polyhedron/demo/Polyhedron/Scene_edit_box_item_config.h

```diff
@@ -0,0 +1,10 @@
+#ifndef SCENE_EDIT_BOX_ITEM_CONFIG_H
+#define SCENE_EDIT_BOX_ITEM_CONFIG_H
+
+#ifdef SCENE_EDIT_BOX_ITEM_EXPORTS
```

Comment:
The name of the target is `scene_edit_box_item`, in lower case, so that line should be:
```C++
#ifdef scene_edit_box_item_EXPORTS
```